### ISSUE TITLE
Add to F-Droid repo

### ISSUE THREAD

**Issue by bt4ibwem8** [theothernt/AerialViews#133]
Hello,
what about to make f-droid build? It would be cool, right?

**Comment by theothernt** [comment]
Yes, it would. I can hopefully add it to F-Droid on the next release - which might be in a few days or so.

Do you have the F-Droid store/app installed on your phone or TV and prefer to install and update it that way?

**Comment by bt4ibwem8** [comment]
Yes, have f-droid client installed on TV box. I have degoogled both phone and TV.

**Comment by bugfest** [comment]
Hi @theothernt! Are you working on this already? I'd be glad to help

**Comment by Retold3202** [comment]
Same here - Please let us know what you need :)

**Comment by theothernt** [comment]
Yes, I would like some help with this @bugfest and @Retold3202.

I have only read a few guides on how to publish to F-Droid, but haven't tested anything yet.

That said, I will have to abstract the Firebase/Logging calls to a helper class, then in an F-Droid 'flavor' of the app, leave those methods empty - does that seem correct?

**Comment by bugfest** [comment]
Yes, exactly. I was tinkering with it yesterday, I think it's feasible without adding complexity. I'll open a PR later today!

**Comment by bugfest** [comment]
Hi @theothernt! PR#165 ready for your review

**Comment by bugfest** [comment]
Thanks for the merge @theothernt. Shall I start preparing the F-Droid manifest for the submission?

**Comment by theothernt** [comment]
> Thanks for the merge @theothernt. Shall I start preparing the F-Droid manifest for the submission?

If you have the time, please do.

**Comment by bugfest** [comment]
I'll open a new issue to track it

**Comment by bt4ibwem8** [comment]
@bugfest 
Why is this stucked?
